src/Scripting/GraphTypeRules.hpp

```
@@ -0,0 +1,187 @@
+            TupleHash
+        > m_ruleTable{};
+
+        std::unordered_set<std::string> m_supportedOps;
+    };
+}
+
+
+namespace Graph {
+    // Lookup table for type combinations of wildcard unary operations: (Type Index, MathOp) -> MathOp(Type Index)::Type
+    // Example: (F64, Negate) -> F64
+    inline Impl::UnaryMathTypeRules UnaryMathTypeRules;
```

Clang (14.0.6)

[warning] variable 'UnaryMathTypeRules' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)